gamemode/modules/f1menu/cl_searchbox.lua

```diff
@@ -6,15 +6,18 @@ function PANEL:Init()
     self.BaseClass.Init(self)
     self.F1Down = true
     self:SetFont("DarkRPHUD2")
-    self:SetTextColor(Color(255,255,255,255))
-    self:SetCursorColor(Color(255,255,255,255))
 
     self.lblSearch = vgui.Create("DLabel", self)
     self.lblSearch:SetFont("DarkRPHUD2")
-    self.lblSearch:SetColor(Color(200, 200, 200, 200))
     self.lblSearch:SetText(DarkRP.getPhrase("f1Search"))
     self.lblSearch:SizeToContents()
     self.lblSearch:SetPos(5)
+
+    if GAMEMODE.Config.DarkRPSkin == "DarkRP" then
+        self:SetTextColor(Color(255,255,255,255))
+        self:SetCursorColor(Color(255,255,255,255))
+        self.lblSearch:SetColor(Color(200, 200, 200, 200))
+    end
```

Comment:
There is a much better was to do this. I have it in my copy but haven't had the chance to send a pull request for it.

You could remove the first two colour setting lines and replace them with

```lua
SKIN.colTextEntryText       = Color(255, 255, 255)
SKIN.colTextEntryTextCursor = Color(255, 255, 255)
```

in the skin.

As for the `lblSearch`, because it only applies to _this_ label then it's a bit more awkward. There's a GMod pull request open https://github.com/garrynewman/garrysmod/pull/972 that standardises placeholders (I've requested changes for proper SKIN support) but don't expect it to be merged in the next couple years. For now, I'd probably just create a custom SKIN field for this and call `SetColor` if it exists though it will unfortunately be something we'd need to provide backwards compatibility for if placeholders did arrive properly in GMod.